cmd/disk-cache-backend.go

```diff
@@ -305,12 +304,24 @@ func (c *diskCache) purge(ctx context.Context) {
 		if err != nil || (fi.ModTime().Before(expiry) && len(cachedFiles) == 0) {
 			removeAll(cacheDir)
 			scorer.adjustSaveBytes(-fi.Size())
-			continue
+			// Proceed to next file.
+			return errSkipFile
```

Comment:
What is the point of returning `errSkipFile` - isn't this just normal operation?
```suggestion
			return nil
```